script/casperquiz.js

```diff
@@ -1,7 +1,5 @@
 function validate(event) {
     
-    event.preventDefault();
```

Comment:
If your alerts don't show anymore, you can add back this line, but after the alerts, before every "return False"